drivers/axi_core/spi_engine/spi_engine.c

```diff
@@ -582,7 +582,7 @@ static int32_t spi_engine_transfer_message(struct spi_desc *desc,
 		for(i = 0; i < desc_extra->offload_tx_len; i++)
 			spi_engine_write(desc_extra,
 					 SPI_ENGINE_REG_OFFLOAD_SDO_MEM(0),
-					 bswap_constant_32(msg->tx_buf[i]));
+					 msg->tx_buf[i]);
```

Comment:
Please use the first (at least) 7 characters of the commit in the commit message.
4d73dcf

Github search cannot find 69b2cc556ecf for example because it's from the end of the hash and it's not standard. I was able to find it only with git command line.